packages/cms-cli/package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/HubSpot/hubspot-cms-tools"
   },
   "dependencies": {
-    "@hubspot/api-auth-lib": "^1.1.7",
     "@hubspot/cms-lib": "^2.0.0-beta.2",
     "commander": "^2.16.0",
+    "express": "^4.17.1",
     "inquirer": "^6.3.1",
     "open": "^7.0.3",
```

Comment:
This used to be using open@6.4.0 in cms-lib.  Are there any breaking changes we are concerned with going to 7.0.3?

Reply:
Good catch, there was a change to use `{ url: true }` for the options. I will update the code .